data/simpy.gram

```diff
@@ -27,9 +27,10 @@ compound_stmt[stmt_ty]: function_def | if_stmt | class_def | with_stmt | for_stm
 
 # NOTE: yield_expression may start with 'yield'; yield_expr must start with 'yield'
 assignment:
-    | !'lambda' a=target ':' b=expression c=['=' d=yield_expression { d }] {
-         _Py_AnnAssign(construct_assign_target(p, a), b, c,
-         a->kind == Name_kind ? 1 : 0, EXTRA) }
+    | '(' a=ann_assign_target ')' ':' b=expression c=['=' d=yield_expression { d }] {
+        _Py_AnnAssign(a, b, c, 0, EXTRA)}
+    | a=ann_assign_target ':' b=expression c=['=' d=yield_expression { d }] {
+        _Py_AnnAssign(a, b, c, 1, EXTRA) }
```

Comment:
If you can do `a: int` and `(a): int` then you can also do `((a)): int` etc. (All of which IMO is stupid and only accepted due to a bug in the grammar but that's backwards compatibility for you.)